Example/UNUMColorPicker/ViewController.swift

```diff
@@ -15,41 +15,39 @@ class ViewController: UIViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view, typically from a nib.
 
         //color data
         let colorData: [UIColor] = [.white, .black, .blue, .green, .yellow, .red, .cyan, .brown]
 
```

Comment:
The main change I made was changing the View to a ViewController. This gets us to what was originally decided, but that's a bit beyond the point. I will discuss this change more in the View file later in the PR.

But as I changed it from a view to a viewController, you'll notice the added `add to view` code below, doing the steps necessary to add a childViewController